shared/locales/en.js

```diff
@@ -388,6 +388,17 @@ Your accounts' balances:`,
                 last_sync: 'last sync on the',
                 new_operations: 'New operations imported during this period:',
                 no_new_operations: 'No new operations have been imported during that period.'
+            },
+            fetch_error: {
+                subject: 'Error when fetching operations',
+                UNKNOWN_WEBOOB_MODULE: `The module is unknown`,
+                NO_PASSWORD: 'The password is not set',
+                INVALID_PASSWORD: 'The password is invalid',
+                EXPIRED_PASSWORD: 'The password expired',
+                INVALID_PARAMETERS: 'The credentials are invalid',
+                GENERIC_EXCEPTION: 'Generic error',
```

Comment:
Please map this one to "unknown error". Ideally, the error message should also be shown in the error report...